src/test/java/com/stripe/functional/SKUTest.java

```diff
@@ -0,0 +1,103 @@
+package com.stripe.functional;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import com.stripe.BaseStripeTest;
+import com.stripe.exception.StripeException;
+import com.stripe.model.DeletedSKU;
+import com.stripe.model.SKU;
```

Comment:
flagging for next major version, are we renaming to Sku? 😱

Reply:
We can make an exception for SKU.